src/library/styles/components/_search.scss

```diff
@@ -1,16 +1,17 @@
-.bsds-search {
-  width: 100%;
-
-  .bsds-input {
-    height: 100%;
-    margin-top: 0;
-  }
+.bsds-field-text-label {
```

Comment:
initially thought this was used elsewhere but we have .bsds-field-label-text and .bsds-field-text-label
thoughts on standardizing or reconsidering naming?

Reply:
actually, looks like this is getting visually hidden anyway. can we just remove?